### DISCUSSION TITLE
OFP on EFB

### DISCUSSION THREAD

**Discussion by andrewgrey715** [flybywiresim/aircraft#4988] [Help] [answered]
When I hit the OFP button, the EFB goes to black screen. I saw a vid where it worked and the flightplan pdf popped up. I am connected to simbrief. Am I doing something wrong? Also, the plane settings are grayed out and I cant select anything.

**Comment by ExampleWasTaken** [comment]
@andrewgrey715 Only LIDO format is supported at the moment.

**Comment by ErickSharp** [comment, marked as answer]
We have now supported all OFP formats with the merger of pull request #5897.
